subprojects/core/src/main/java/org/gradle/api/internal/changedetection/state/ClasspathEntrySnapshotBuilder.java

```diff
@@ -38,7 +38,7 @@
 /**
  * Builds the snapshot of a classpath entry.
  * It can be either used on {@link RegularFileSnapshot}s or {@link ZipEntry}.
- * The {@link NormalizedFileSnapshot}s can be collected by a {@link FileCollectionSnapshotBuilder}.
+ * The {@link NormalizedFileSnapshot}s can be collected by a {@link FileCollectionVisitingSnapshotBuilder}.
```

Comment:
The normalized snapshots are still collected by a `FileCollectionSnapshotBuilder`. See `collectNormalizedSnapshots(FileCollectionSnapshotBuilder builder)`.